bin/run-tests.sh

```diff
@@ -17,22 +17,22 @@ exit_code=0
 for test_dir in tests/*; do
     test_dir_name=$(basename "${test_dir}")
     test_dir_path=$(realpath "${test_dir}")
+    input_path="${test_dir_path}/input/"
     results_file_path="${test_dir_path}/results.json"
     expected_results_file_path="${test_dir_path}/expected_results.json"
 
-    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
+    slug=$test_dir_name
 
-    # OPTIONAL: Normalize the results file
-    # If the results.json file contains information that changes between 
-    # different test runs (e.g. timing information or paths), you should normalize
-    # the results file to allow the diff comparison below to work as expected
-    # sed -i -E \
-    #   -e 's/Elapsed time: [0-9]+\.[0-9]+ seconds//g' \
-    #   -e "s~${test_dir_path}~/solution~g" \
-    #   "${results_file_path}"
+    echo "TESTING: ${test_dir_name}"
+    if [ -f "${test_dir_path}/data.json" ]; then
+        slug=$(cat ${test_dir_path}/data.json | jq -r .slug)
+    fi
+
+    bin/run.sh "${slug}" "${input_path}" "${test_dir_path}" > /dev/null
 
-    echo "${test_dir_name}: comparing results.json to expected_results.json"
-    diff "${results_file_path}" "${expected_results_file_path}"
+    # echo "${test_dir_name}: comparing results.json to expected_results.json"
+    diff -urN "${expected_results_file_path}" "${results_file_path}"
+    rm -f ${results_file_path}
```

Comment:
If this proves a problem for other maintainers it can change in the future.  Personally I'd prefer not to have "build assets" laying around if I can avoid it.  Perhaps you should try `diff -urN`. 😇